any/when_agegroup.md

```diff
@@ -0,0 +1,17 @@
+# when_agegroup
```

Comment:
Using this text for the link is fine, but can you format this so it's more readable? Maybe something like "Date to age group" or something that describes the function to be used.

Reply:
I just realized this doesn't deal with Date fields. It's still numbers...so it could be even more generic, like "Number to category"